README.md

```diff
@@ -236,6 +252,24 @@ open-attestation document-store revoke --network ropsten --address 0x19f89607b52
 ✔  success   Document/Document Batch with hash 0x0c1a666aa55d17d26412bb57fbed96f40ec5a08e2f995a108faf45429ae3511f has been revoked on 0x19f89607b52268D0A19543e48F790c65750869c6
 ```
 
+## Gsn Capable
```

Comment:
not sure about the logic with gsn capable naming, can help to understand ?

Reply:
Erm i just added naming convention to denote that the contract is able to accept relay address and return a specificed paymaster

Reply:
Just like ownable/revokable etc